search/includes/classes/class-queue.php

```diff
@@ -468,6 +470,25 @@ public function offload_indexing_to_queue() {
 		}
 	}
 
+	/**
+	 * Offload term indexing to the queue
+	 */
+	public function offload_term_indexing_to_queue( $term_id ) {
+		$term = \get_term( $term_id );
+
+		if ( is_wp_error( $term ) || ! is_object( $term ) ) {
+			return;
+		}
+
+		if ( $term->count <= self::MAX_SYNC_INDEXING_COUNT ) {
```

Comment:
Is this conditional correct? We want to offload indexing to the queue when the term's object count is _over_ the max sync indexing limit.

Reply:
Yes it's correct. The function called is our original non-cron offloading functionality that works great.

Might need to be renamed or add a comment or something to make it clearer.